pkg/commands/analyse_grafana_test.go

```diff
@@ -38,3 +38,22 @@ func TestParseMetricsInBoard(t *testing.T) {
 	analyse.ParseMetricsInBoard(output, board)
 	assert.Equal(t, dashboardMetrics, output.Dashboards[0].Metrics)
 }
+
+var timeseriesPanelMetrics = []string{
```

Comment:
[nit] Why not just scope this variable inside of `TestParseMetricsInBoardWithTimeseriesPanel`?